### PR TITLE
Branch delete phone param

### PR DESCRIPTION
**Pull request by yashma-sonara**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -49,7 +45,7 @@ public class CommandTestUtil {
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
-    public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
+    public static final String INVALID_PHONE_DESC = " " + "911a"; // 'a' not allowed in phones
```
Comment (Wongqingyee) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:48]: Since we are removing the phone param, I think we dont need this line at all.

**Review by JiaXinEu — CHANGES_REQUESTED** [review]
- Perhaps you might want to edit the files under `test/data/JSONAddressBookStorageTest` and `test/data/JSONSerializableAddressBookTest` to remove any phone parameter. 
- `JSONAdressBookStorageTest/invalidAndValidPersonAddressBook.json` is meant to consist of a valid and an invalid person, but in the master branch the data is invalid due to the phone number which has been removed in this PR. Maybe replacing it with another invalid parameter will help with passing the test.

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 2]
```diff
@@ -168,31 +155,25 @@ public void parse_multipleRepeatedFields_failure() {
...
-        // valid followed by invalid
-        Index targetIndex = INDEX_FIRST_PERSON;
-        String userInput = targetIndex.getOneBased() + INVALID_PHONE_DESC + PHONE_DESC_BOB;
-
-        assertParseFailure(parser, userInput, Messages.getErrorMessageForDuplicatePrefixes(PREFIX_PHONE));
 
-        // invalid followed by valid
-        userInput = targetIndex.getOneBased() + PHONE_DESC_BOB + INVALID_PHONE_DESC;
+        Index targetIndex = INDEX_FIRST_PERSON;
+        String userInput = targetIndex.getOneBased() + INVALID_PHONE_DESC;
```
Comment (Wongqingyee) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:160]: I think we dont need this line.
Can just declare the String userInput in line 164.

**Review by JiaXinEu — APPROVED** [review]

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 3]
```diff
@@ -51,7 +50,7 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
      */
     public JsonAdaptedPerson(Person source) {
         name = source.getName().fullName;
-        phone = source.getPhone().value;
+        //phone = source.getPhone().value;
```
Comment (Wongqingyee) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java:53]: I think can just delete it instead of commenting it.

**Comment by Wongqingyee** [comment]
I think can delete the Phone.java and PhoneTest.java too

**Comment by JiaXinEu** [comment]
Seems like there are conflicts to be resolved before merging

**Pull request merged**